clusterloader2/pkg/measurement/util/informer/informer.go

```
@@ -61,7 +61,7 @@ func NewDynamicInformer(
 func addEventHandler(i cache.SharedInformer,
 	handleObj func(interface{}, interface{}),
 ) {
-	i.AddEventHandler(cache.ResourceEventHandlerFuncs{
+	_, _ = i.AddEventHandler(cache.ResourceEventHandlerFuncs{
```

let's not ignore error - let's at least log an error in such case